app/src/main/java/org/mozilla/fenix/settings/AccessibilityFragment.kt

```diff
@@ -25,6 +26,23 @@ class AccessibilityFragment : PreferenceFragmentCompat() {
         (activity as AppCompatActivity).title = getString(R.string.preferences_accessibility)
         (activity as AppCompatActivity).supportActionBar?.show()
 
+        val forceZoomPreference = findPreference<SwitchPreference>(
+            getPreferenceKey(R.string.pref_key_accessibility_force_enable_zoom)
+        )
+        forceZoomPreference?.isVisible = FeatureFlags.forceZoomPreference
+
+        if (FeatureFlags.forceZoomPreference) {
+            forceZoomPreference?.setOnPreferenceChangeListener<Boolean> { preference, shouldForce ->
+                val settings = preference.context.settings()
+                val components = preference.context.components
+
+                settings.forceEnableZoom = shouldForce
+                components.core.engine.settings.forceUserScalableContent = shouldForce
```

Comment:
Should we also reload the open session(s) so that this new engine setting takes effect? (see `textSizePreference`)

Reply:
@ekager I thought so as well, but I tested it without reloading and it worked!